tests/fixtures/blast/blastSettingsConfig.json

```diff
@@ -802,6 +802,13 @@
     "tblastn": ["matrix", "gapopen", "gapext"],
     "blastp": ["matrix", "gapopen", "gapext"]
   },
+  "valid_sensitivities_for_program": {
```

Comment:
Why do we need this? Can't this be calculated from the keys of the `presets.settings` object?

Reply:
I went by what we did with `valid_parameters_for_program` as it would be easier to access it, though now think it is more or less redundant. But this also means we do the same for ` valid_parameters_for_program` right?

Reply:
Made the changes for this ✅